exercises/atbash-cipher/.meta/solutions/atbash_cipher.rb

```diff
@@ -1,16 +1,24 @@
-class Atbash
+class AtbashCipher
```

Comment:
Didn't follow the proper convention before.

Reply:
I thought `Atbash` is fine, as it is a cipher, Like saying `GuygastineauPerson` when we already know you are a person or `MiamiCity` when we know it is a city.

Reply:
I was under the impression that it is a hard requirement of this track for class names to be the CamelCase equivalent of the exercise slugs.

At least that is what was said in the `armstrong-numbers` exercise when it was brought up that `ArmstrongNumber` would fit better...

Reply:
It is a style requirement that it is CameCase, but I wasn't positive that it had to exactly match the slug... though I was aware that it usually does.

Reply:
Confirmed that it doesn't have to, it will be delivered, etc.  But if we want to, that is fine as well.  No strong feeling about it being a perfect match, and I think it is named well as it is.

Anyone else have strong feelings about this change either way?

An example of where this may change in the future may be the "Grains" method, as well as potentially "Series".  The slug may not end up matching the class name, or even the file name(s).  The slug identifies the exercise...

Reply:
So, it should stay as is in the PR or revert to Atbash?

Reply:
I would leave it as it was... (there wasn't a request to change it, and it won't be confusing when I update the exercise and my program doesn't fail all the tests due to the update).

Reply:
Cool. That is what I thought you meant.

What do you think about my suggestions for changing `#convert`?

Reply:
It is fine.  No strong opinion on the change, except that it doesn't belong in this PR.  The changes should be only to let the solution pass the tests as they were generated.  Other changes aren't really relevant to the PR.

They can happen here, but I also don't have strong feelings about the example solutions being anything other than a proof that the exercise can be written in the track.  It doesn't have to be exemplary, as that isn't its purpose.

Reply:
Right, well I'm sorry I did it in my last commit.  I opened the solution to revert the class name, and my OCD just got the better of me.

Reply:
Emacs EnhRuby mode was freaking out about the 1 letter argument names too :rofl:

Reply:
I reread the first message in the PR, and the intention is to complete the solution as well, as it was "half implemented" so the solution does belong here, though the scope of the PR has grown, since I thought you were only doing the generator, rather than "also the half implementation".

Since the title was only implemented generator, that is what is persisted at the top (title) of the page viewer, and that is what my review was based on.